modules/autocomplete-picker/test/spec.coffee

```diff
@@ -24,15 +29,22 @@ describe 'autocomplete-picker', ->
       callback(trivialItems)
     setTimeout(cb, trivialAsyncWait)
 
+  before ->
+    clock = sinon.useFakeTimers()
+    hx.loop = mock_hx_loop
```

Comment:
The `open autocomplete` tests relied on the completion of the dropdown animation that was not correctly processed in the tests when using `useFakeTimers` so I copied the mock loop from the dropdown tests